src/toolkit/gis.h

```diff
@@ -0,0 +1,158 @@
+#ifndef CYCLUS_SRC_GIS_H_
+#define CYCLUS_SRC_GIS_H_
+
+#include <math.h>
+#include <stdio.h>
+#include <sstream>
+#include <string>
+
+#include "cyclus.h"
+
+// forward declarations
+namespace cyclus {
+namespace toolkt {
+class GIS;
+}
+}
+
+namespace cyclus {
+namespace toolkit {
+/// @class GIS
+/// The GIS class is a basic class that stores the geographic location of each
+/// agent in latitude and longitude and follows the ISO
+/// 6709 standard. Allows string expression of GIS objects as specified in
+/// ANNEX H of ISO 6709 found here: 'https://www.iso.org/standard/39242.html'
+/// or a quick simplified version found here
+/// 'https://en.wikipedia.org/wiki/ISO_6709'
+///
+/// Brief Description of ISO 6709:
+/// 1. Latitude comes before Longitude
+/// 2. North latitude is positive
+/// 3. East longitude is positive
+/// 4. Fraction of degrees (decimal values) is prefered in digital data
+/// exchange.
+///
+/// The class is adapted from 'https://github.com/jaime-olivares/coordinate'
+/// under the MIT License.
+///
+/// Longitude and Latitude is stored as seconds of degrees. This was also
+/// adapted from 'https://github.com/jaime-olivares/coordinate'. This, according
+/// to the jaime-olivares, allows the coordinate elements such as degrees,
+/// minutes, and seconds to "remain on the integral portion of values, with the
+/// exception of decimal of seconds, avoiding loss of precision." This is
+/// calculated by multiplying decimal degrees by 3600.
+/// example: 05.2169 -> 18780.84
+
+class GIS {
+ public:
+  /// The default constructor for GIS. This will creat an object with
+  /// lat =0, and long=0.
+  GIS();
+
+  /// GIS constructor with latitude and longditude as degrees as a decimal
+  /// number.
+  /// @param deciaml_lat latitude expressed in degrees as a decimal number.
```

Comment:
This is perhaps overly nitpicky, so forgive me in advance, but the number here is not decimal.  Rather the computer uses  binary, which is what is being used when you use `float`.  If you to meant to say real numbers here instead, that is not strictly correct either as floating point is an approximation to the reals. 

I would simply say floating point instead of decimal.